docs/user-manual/civiform-admin-guide/manage-questions.md

```diff
@@ -33,6 +33,21 @@ Input: 123-45-6789 Example obfuscated output: d158596dd5a6cae6fcb282832885631553
 
 Input: 123-45-6780 Example obfuscated output: 01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b
 
+### Universal questions
+
+Universal questions are questions that are intended to be used by all programs. When a question is marked as universal, it appears at the top in a separate section in the questions tab and the question bank. Any question may be set as a universal question.
+
+![](../../.gitbook/assets/universal-question.png)
+![](../../.gitbook/assets/universal-questions-tab.png)
+
+On the programs tab, each program notes how many of the universal questions are used in the program. 
+
+![](../../.gitbook/assets/universal-questions-program-list.png)
```

Comment:
suggestion (non-blocking): add a red box or arrow to indicate where in the programs view it shows you how many universal questions are used.